.github/workflows/check.yml

```diff
@@ -41,3 +41,5 @@ jobs:
           | jq '.packages | .[] | .name' \
           | rg --invert-match 'atleta-(node|runtime)' \
           | xargs -I {} cargo clippy --package {} --all-targets --all-features -- --deny warnings --deny missing_docs
+    - name: Run shellcheck
+      run: find "$GITHUB_WORKSPACE/deploy" "$GITHUB_WORKSPACE/scripts" -name '*.sh' -print0 | xargs -0 -I {} shellcheck --exclude=SC1090 --exclude=SC1091 {}
```

Comment:
We need to check not only the `deploy` dir, but all scripts in the project (`scripts` dir also, as well as any further updates).
The `.sh` extension IMO is also not the best mark of shell scripts since 1) it can be omitted 2) it is confusing since scripts are actually bash scripts, not sh, and more generally depend on [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) and not from file extension.

So here the proposed solution: replace `find` by [`fd-find`](https://github.com/sharkdp/fd) (well-known utility for any rust-lang user) and then we can find all executables `fd --type executable`. Here the some executable `.ts` script, but I guess it should be fixed as `chmod a-x …`. Probably the same files list can be built with vanilla `find`, but I'm not sure.

<img width="487" alt="image" src="https://github.com/user-attachments/assets/ac2bf55f-eb3d-468b-9521-4150c6e419c4">

Reply:
With for and find also works :)
`for file in $(fd --type executable); do shellcheck --exclude=SC1090 --exclude=SC1091 "$file"; done`

Reply:
It depends on which version of the `find` we are using.
Like on macOS we do not have some flags that are available on modern linux.
<img width="368" alt="image" src="https://github.com/user-attachments/assets/22aa4d47-b97b-4f90-a2ef-4a02afc481ba">